### PR TITLE
DOC-9070: Release Notes for 7.0.1

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

File: modules/release-notes/pages/relnotes.adoc
Comment (amarantha-k) [inline modules/release-notes/pages/relnotes.adoc]: We list the latest release at the top of the page, so release 7.0.1 should be before 7.0.0

File: modules/release-notes/pages/relnotes.adoc
Comment (amarantha-k) [inline modules/release-notes/pages/relnotes.adoc]: v7.0 was a major release and hence did not need the fixed issues listed explicitly. For the maintenance releases, we do need to explicitly list the fixed issues. Pls see v6.5.x release notes for reference

**Review by amarantha-k — CHANGES_REQUESTED** [review]
Added comments on the PR

File: modules/release-notes/pages/relnotes.adoc
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc]: I"ve changed the order accordingly. Thanks.

File: modules/release-notes/pages/relnotes.adoc
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc]: I've removed the reference to the filter.

**Review by ritamcouchbase — APPROVED** [review]

**Review by amarantha-k — APPROVED** [review]
looks good from editorial perspective. Waiting on QE review/sign-off.

**Review by arod1987 — COMMENTED** [review]
looks good to me

File: modules/release-notes/pages/relnotes.adoc
Comment (mihirkamdarqe) [inline modules/release-notes/pages/relnotes.adoc:26]: This should be reworded to say something like "Fixed an error occurring when running Flex index queries on a setup that uses alternate addresses"

**Pull request merged**
